addons/core/functions/fnc_playerJipHint.sqf

```diff
@@ -0,0 +1,40 @@
+#include "script_component.hpp"
+/*
+ * Author: PabstMirror
+ * Handles player jipping, sends hint on last group
+ */
+
+
+if (isServer) then {
+    GVAR(playerMap) = createHashMap;
+    [QGVAR(playerConnected), {
+        params ["_player", "_profileName"];
+        private _msg = if (_profileName in GVAR(playerMap)) then {
+            (GVAR(playerMap) get _profileName) params ["_lastSide", "_lastGroup"];
+            private _sideText = ["?", "B", "O", "I"] select ([blufor, opfor, independent] find _lastSide);
+            format ["has JIPed; last in [%1-%2]", _sideText, _lastGroup]
+        } else {
+            private _lastSide = side group _player;
+            private _lastGroup = groupId group _player;
+            GVAR(playerMap) set [_profileName, [_lastSide, _lastGroup]];
+            format ["has JIPed"];
+        };
+        INFO_2("playerConnected: %1: [%2]",_profileName,_msg);
+        if (CBA_missionTime < 60) exitWith {};
```

Comment:
future:
can  probably set this timer to be the safe-start length as specified in the mission parameters option